tests/e2e/integration/Explorer/main.js

```diff
@@ -0,0 +1,193 @@
+import homepage from '../../support/pages/homepage';
+import blockheight from '../../support/pages/blockheight';
+import devices from '../../support/devices';
+
+devices.forEach((device) => {
+  describe('Explorer autotest', () => {
```

Comment:
I think "autotest" is not necessary here. `describe()` should tell us about the feature under test, not about the test